### PR TITLE
feat(applications-service): introduce page header middleware and upda…

### PR DESCRIPTION
**Pull request by emil-placheta**
…te header,update 2 journeys

https://pins-ds.atlassian.net/jira/software/c/projects/ASB/boards/166?selectedIssue=ASB-826

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/config.js
Comment (emil-placheta) [inline packages/forms-web-app/src/config.js:1]: not 100% sure if this is the best place for those

File: packages/forms-web-app/src/middleware/get-header-title.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { getHeaderTitleFromPath } = require('../utils/get-header-title-from-path');
+
+async function setHeaderTitle(req, res, next) {
```
Comment (emil-placheta) [inline packages/forms-web-app/src/middleware/get-header-title.js:3]: why async

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**